proposal.rst

```diff
@@ -369,6 +369,9 @@ Plan :
         with exceptions, they are panic defer and recover.A Panic is similar to an exception which can occur an runtime exception.
         C++ exceptions can be handled in go from swig wrappers as follows(https://github.com/srinivasyadav18/xapian-gsoc-plan/blob/master/example.i#L16).
         Which ever class function throws an exception in c++ , the wrapped function in Go returns the error as value.
+        Database error handling - https://github.com/srinivasyadav18/xapian-gsoc-plan/blob/master/go.i#L77
+        https://github.com/srinivasyadav18/xapian-gsoc-plan/blob/master/main.go#L36
+        error (pastebin) - https://pastebin.com/AuFpiRdQ
```

Comment:
Please include content in your proposal rather than linking to external sites.  The final proposal you submit is essentially what you're saying you'll do if chosen, so it ought to be self-contained rather referring out to information on external sites can change or even disappear (content on paste sites typically expires after a while so is particular poor to refer to).

It would also be better to inline or include the content referred to in https://github.com/srinivasyadav18/xapian-gsoc-plan/blob/master/main.go for the same reasons.  That's at least that's under version control, so not quite as bad as linking to a paste site.

Reply:
> Please include content in your proposal rather than linking to external sites. The final proposal you submit is essentially what you're saying you'll do if chosen, so it ought to be self-contained rather referring out to information on external sites can change or even disappear (content on paste sites typically expires after a while so is particular poor to refer to).
> 
> It would also be better to inline or include the content referred to in https://github.com/srinivasyadav18/xapian-gsoc-plan/blob/master/main.go for the same reasons. That's at least that's under version control, so not quite as bad as linking to a paste site.

Is it fine the proposal will get longer? I have include the code now in proposal . If it is not good please give a feedback. I have put even the links at the end in references.

Reply:
Don't worry about length (at least if the content is relevant as this is).